explorer/src/components/Header/index.tsx

```diff
@@ -34,14 +34,14 @@ export const Header: React.FC<HeaderProps> = ({ isOpened, setIsOpened }) => {
     <header className="px-5 md:px-14 xl:px-[60px] py-3 justify-between items-center inline-flex">
       <div className="justify-start items-center gap-6 flex self-stretch">
         <Link to={APP_ROUTES.HOME} className="shrink-0 hover:opacity-70">
-          <img src={logo} className="h-6 xl:h-9 cursor-pointer" alt="Verax logo" />
+          <VeraxLogo />
```

Comment:
You need to add the styles that were. As the screen width changes, the logo increases and decreases.
Also, check all the SVGs that you changed.